README.md

```diff
@@ -13,6 +13,11 @@ pip install maskpass
 
 It is currently recommended to enforce a version if you have plans to use it in a project, since backward incompatible changes may occur.
 
+If the installation returns an error, it may be worth trying to install the Python development packages.
```

Comment:
I would prefer a less ambiguous message like:  
_If the installation show the following error: `fatal error: Python.h: No such file or directory`, you need to install the Python development packages._  
and then the apt command for installation